src/styles/card.scss

```diff
@@ -0,0 +1,99 @@
+.product-card {
+  display: flex;
+  flex-direction: column;
+  box-sizing: border-box;
+  width: 100%;
+  max-width: 200px;
+  margin-bottom: 50px;
+  border-radius: 5px;
+  border: 1px solid $border-color;
+  background: $back-color;
+
+  &__overview {
+    padding: 16px;
+  }
+
+  &__icon {
+    display: flex;
+    margin: 32px 19px 24px;
+    width: 160px;
+    height: 134px;
+  }
+
+  &__name {
+    margin: 0;
+    margin-bottom: 4px;
+    text-align: left;
+    font-size: 12px;
+    font-weight: 500;
+    line-height: 18px;
+    color: $main-color;
+  }
+
+  &__coding {
+    margin-bottom: 16px;
+    text-align: left;
+    font-size: 10px;
+    font-weight: 400;
+    line-height: 14px;
+    color: $addition-color;
+  }
+
+  &__rating {
+    display: flex;
+    margin-bottom: 24px;
+    align-items: center;
+    justify-content: space-between;
+    color: $main-color;
+  }
+
+  &__review {
+    text-align: right;
```

Comment:
the same question(do u need it? also in all the cases)

Reply:
I'll check it too, thanks.